File: src/main/java/no/nav/fo/veilarb/dokument/domain/DokumentbestillingDto.java
package no.nav.fo.veilarb.dokument.domain;

import lombok.Value;
import lombok.experimental.Accessors;
import no.nav.common.types.identer.EnhetId;
import no.nav.common.types.identer.Fnr;

import java.util.List;

@Value
@Accessors(fluent = true)
public class DokumentbestillingDto {
    Fnr brukerFnr;
    MalType malType;
    EnhetId enhetId;
    String begrunnelse;
    List<String> opplysninger;
}

File: src/main/java/no/nav/fo/veilarb/dokument/domain/DokumentbestillingDto.kt
package no.nav.fo.veilarb.dokument.domain

import no.nav.common.types.identer.EnhetId
import no.nav.common.types.identer.Fnr

data class DokumentbestillingDto(
        val brukerFnr: Fnr,
        val malType: MalType,
        val enhetId: EnhetId,
        val begrunnelse: String?,
        val opplysninger: List<String>
)
